entproto/extension.go

```diff
@@ -66,6 +67,16 @@ func WithProtoDir(dir string) ExtensionOption {
 	}
 }
 
+// WithProtoHelper sets whether to generate a helper file next to each .proto file.
+// The helper file contains a //go:generate directive to invoke protoc and compile Go code from the protobuf definitions.
+// If generate.go already exists next to the .proto file, this step is skipped.
+// Set generate to false to disable the generation of the generate.go file.
+func WithProtoHelper(generate bool) ExtensionOption {
+	return func(e *Extension) {
```

Comment:
In terms of API I prefer "SkipGenFile" , no args. As we generate the file by default. Also "helper" is too ambiguous, IMO

Reply:
updated